src/innhold/innhold-logikk-niva4.tsx

```diff
@@ -24,29 +24,27 @@ interface StateProps {
     fremtidigSvar: FremtidigSituasjonSvar | null;
     foreslattInnsatsgruppe: ForeslattInnsatsgruppe | undefined | null;
     reservasjonKRR: boolean;
+    servicegruppe: Servicegruppe | null;
     opprettetRegistreringDato: Date | null;
     harEgenvurderingbesvarelse: boolean;
     egenvurderingbesvarelseDato: Date | null;
 }
 
 const InnholdLogikkNiva4 = ({
                                 erSykmeldtMedArbeidsgiver, fremtidigSvar, harEgenvurderingbesvarelse,
-                                egenvurderingbesvarelseDato,
-                                foreslattInnsatsgruppe, reservasjonKRR, opprettetRegistreringDato
+                                egenvurderingbesvarelseDato, foreslattInnsatsgruppe, reservasjonKRR, servicegruppe,
```

Comment:
Hvorfor heter det `foreslattInnsatsgruppe`og ikke `foreslattServicegruppe`her?

Reply:
Rett og slett mangel på standardisering innad i FO 🙄

Reply:
https://nav-it.slack.com/archives/C6YT8EW6Q/p1562843652124300